adyenv6b2ccheckoutaddon/acceleratoraddon/web/webroot/_ui/responsive/common/js/adyen.checkout.js

```diff
@@ -76,6 +76,8 @@ var AdyenCheckoutHybris = (function () {
 
                 if ( isInvalidCard ) {
                     window.alert( 'This ' + this.getCardType() + ' card is not allowed' );
```

Comment:
@rkewlani @martinsrenato Do you think we should change this message to fill all the details instead?

Reply:
I think so.
It can be invalid because of missing fields, like you said, but also because the card number typed is invalid or from an unsupported brand. So we could have specific messages for both cases, if there's any way to identify that through the component validation, or a generic message that works for both.

Reply:
For Unsupported cards it shows message in the component itself. 

![Screen Shot 2021-02-18 at 3 33 27 PM](https://user-images.githubusercontent.com/16626736/108372105-c07d4880-71fe-11eb-9367-f60734d3aa9e.png)

So i think we can show a generic message like "Please check your card details" for the whole form.

Reply:
I agree

Reply:
Message is updated now.